source/api/ut.pkb

```diff
@@ -103,7 +103,9 @@ create or replace package body ut is
       raise e_package_invalidated;
     end if;
   end;
-
+  
+  /* Testing of PR sonar*/
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This single line comment should use the single line comment syntax "--" ([plsql:SingleLineCommentsSyntaxCheck](https://sonarcloud.io/organizations/utplsql/rules#rule_key=plsql%3ASingleLineCommentsSyntaxCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=utPLSQL&pullRequest=711&issues=AWSQNHcNl6vpliCebOhK&open=AWSQNHcNl6vpliCebOhK)

[ISSUE_KEY]: AWSQNHcNl6vpliCebOhK